lapis2/src/test/kotlin/org/genspectrum/lapis/model/SiloFilterExpressionMapperTest.kt

```diff
@@ -583,8 +583,8 @@ class SiloFilterExpressionMapperTest {
                     ),
                     And(
                         And(
-                            NucleotideSymbolEquals(null, 300, "G"),
                             NucleotideSymbolEquals(null, 400, "A"),
+                            NucleotideSymbolEquals(null, 300, "G"),
```

Comment:
Do we care that the order which is sent to SILO is reversed from the input?

Reply:
The order should not matter.